modules/redpanda/redpanda_test.go

```diff
@@ -698,3 +705,23 @@ func TestRedpandaBootstrapConfig(t *testing.T) {
 		require.False(t, needsRestart)
 	}
 }
+
+func detectContainerHostAddress(image string, ctx context.Context) (string, error) {
```

Comment:
suggestion: pass in the test so you can use require, eliminating the need to return an error.

Reply:
Done